infra/cmake/modules/IdentifyPlatform.cmake

```diff
@@ -51,6 +51,8 @@ endif()
 
 if("${TARGET_ARCH}" STREQUAL "x86_64")
   set(TARGET_ARCH_BASE ${TARGET_ARCH})
+elseif("${TARGET_ARCH}" STREQUAL "armv8-m")
+  set(TARGET_ARCH_BASE "arm")
```

Comment:
Arm says, `ARMv8-M: Supports only a subset of the T32 instruction set.`
- https://developer.arm.com/Architectures/M-Profile%20Architecture